services/todo-legacy/README.md

```diff
@@ -0,0 +1,39 @@
+# Todo Application with Legacy Juggler
+
+## Setup
+
+### Basic Usage
+To use this example:
+```
+# Install typescript and ts-node
+npm i -g typescript ts-node
```

Comment:
Why is it necessary to install `typescript` and `ts-node` globally? IMO, they should be regular dependencies of this project. Remember, `npm` adds `$PWD/node_modules/.bin` to `$PATH` when running package scripts.

IMO, the instructions should be simplified as follows:

````
# Install dependencies
npm i
# Go!
npm start
```